tests/serial/map-if-necessary/three-solvers/mixed-substeps/Multi.cpp

```diff
@@ -0,0 +1,31 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(ThreeSolvers)
+BOOST_AUTO_TEST_SUITE(MixedSubsteps)
+
+BOOST_AUTO_TEST_CASE(Multi)
+{
+  PRECICE_TEST("A"_on(1_rank), "B"_on(1_rank), "C"_on(1_rank));
+
+  std::vector<int> readMappings{2, 0, 2, 0, 2, 0};
```

Comment:
I am not sure I understand the zeros here. Could you comment?

Reply:
I think the zeros are wrong. There should be new data from A and B also in the second iteration.
```suggestion
  std::vector<int> readMappings{2, 2, 2, 2, 2, 2};
```